test/index-spec.js

```diff
@@ -25,7 +25,7 @@ describe("Escape carrigage return", function() {
       "1\r"
     ].join("");
 
-    var output = ["hasrn\n", "hasn\n", "\n", "hellof\n", "123\r"].join("");
+    var output = ["hasrn\n", "hasn\n", "\n", "hellof\n", "123"].join("");
```

Comment:
I believe that it shouldn't leave the `\r` at the end. Is that right, @lgeiger? If it was OK, it's quite simple to modify the new code though to keep the old behavior.

Reply:
That is a good question. It has been some time since I looked at this code, but if I recall correctly this was intended.
I think the reasoning was that processing a string with this package should keep carriage returns in the last line in case the frontend would like to handle them in a special case. E.g. display a curser at this position similar to how a proper terminal emulator would do it.

That being said, I am not sure if this behaviour is actually handled correctly for all cases or whether this is just an artefact of this test. Would it be problematic for you if we'd leave the `\r`  at the end here?

Reply:
Makes sense! It will be really and shouldn't be a problem, so I will do it soon